README.md

```diff
@@ -0,0 +1,103 @@
+<a name="readme-top"></a>
+
+
+<div align="center">
+  
+
+  <h3><b>To Do List App</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[To Do List App \] ](#-to-do-list-app--)
+  - [🛠 Built With **HTML, CSS and JavaScript**](#-built-with-html-css-and-javascript)
+    - [Tech Stack **HTML, CSS and JavaScript**](#tech-stack-html-css-and-javascript)
+    - [Key Features ](#key-features-)
+    - [LiveDemo](#livedemo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Project\_presentation](#project_presentation)
+  - [🔭 Future Features ](#-future-features-)
+  - [👥 Authors ](#-authors-)
+  - [Contributions](#contributions)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 [To Do List App ] <a name="about-project"></a>
+**To Do List App** is a project under microverse Module 2 block 2 aimed at testing my understanding of ES6 syntax, modules and webpack. This app takes and stores a list of activities to be done by a person.
+## 🛠 Built With **HTML, CSS and JavaScript**
+
+### Tech Stack **HTML, CSS and JavaScript**
+### Key Features <a name="key-features"></a>
+- **Contains To Do List App project that takes and stores to do items as ojects in an array**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### LiveDemo
+[Live Demo] You can see the live Demo of the website here 
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: A web browser.
+### Setup
+
+Clone this repository to your desired folder:
+cd Module_1 
+-git clone 
+
+### Install
+
+_No installation is required, double click the index.html file to open it.
+
+### Usage
+
+To run the project, execute the following command:
+-Double click the index.html file.
+
+### Project_presentation
+[Project Presentation] You can see the live presentation of the key concepts taken into account while creating this project here: 
+## 🔭 Future Features <a name="future-features"></a>
```

Comment:
- [ ] Great job on setting up webpack and nailing the styling of the app! Just wanted to mention that **it would be fantastic if you could update your `README` with the steps to start this new webpack project**. This way, anyone can easily understand and launch your project without encountering any issues. A helpful hint would be to include commands like "npm install" and "npm start." Keep up the awesome work!